src/app/beer_garden/garden.py

```diff
@@ -57,6 +58,33 @@ def local_garden() -> Garden:
     )
 
 
+def sync_garden(garden):
+
+    if garden == config.get("garden.name"):
+        publish(
+            Event(
+                name=Events.GARDEN_UPDATED.name,
+                payload_type="Garden",
+                payload=Garden(
+                    name=garden,
+                    status="RUNNING",
+                    systems=get_systems(),
+                    namespaces=get_namespaces(),
+                ),
+            )
+        )
+    else:
+        router.forward(
+            Operation(operation_type="GARDENS_SYNC", target_garden_name=garden)
+        )
```

Comment:
This is an issue - the router module should be the only module making routing decisions. Pushing that logic down into the actual functions is not going to scale.

Reply:
Moved this up into a routing function in `route.py` function. Went back and forth in doing everything here, but I restricted it to only the logic on where to execute the function, not executing the function.